### DISCUSSION TITLE
View SARIF reports when there are no GHAS license

### DISCUSSION THREAD

**Discussion by dimapasko** [reposaur/reposaur#48] [General]
I configured GitHub action with `reposaur/repo-audit-action@main` to check all my organization repositories.
GitHub action output says:

> <nil> INF Wrote 12 reports to /home/runner/work/idp-github-policies/idp-github-policies/.reposaur
> <nil> INF Uploaded 12 reports to GHAS

We have enterprise licencse but there are no GHAS license available(all repos are internal).

Could you give piece of advice how could i view this reports?
Should i have GHAS license for this?
Maybe there are some workarounds.

**Comment by dimapasko** [comment]
May i suggest an option to get report as job summary
https://github.blog/2022-05-09-supercharging-github-actions-with-job-summaries if GHES is not enabled

**Reply by crqrdotcom** [reply]
Hey @dimapasko - sorry for late reply I have some issue with my GitHub notifications. I really think that for now that's the only option you've without having GHAS. Thanks for the suggestion.

I'll look into it over this week and try to implement it in the action and let you know. 

BTW, you can still have GHAS for internal repositories if you have GitHub Enterprise Cloud or GitHub Enterprise Server (see https://docs.github.com/en/billing/managing-billing-for-github-advanced-security/about-billing-for-github-advanced-security).

**Reply by dimapasko** [reply]
It would be great to have a fallback scenario in case you don't have access to GHAS. 

We currently have GitHub Enterprise Cloud licenses but GHAS is an additional cost, so we don't have it yet.
